tests/update_reference_data.sh

```diff
@@ -38,6 +38,7 @@ tests[spe1oilgas]="flow spe1 SPE1CASE2_OILGAS spe1_oilgas"
 tests[spe1nowells]="flow spe1 SPE1CASE2_NOWELLS spe1_nowells"
 tests[spe1thermal]="flow spe1 SPE1CASE2_THERMAL spe1_thermal"
 tests[ctaquifer_2d_oilwater]="flow aquifer-oilwater 2D_OW_CTAQUIFER ctaquifer_2d_oilwater"
+tests[fetkovich_2d]= "flow aquifer-fetkovich 2D_FETKOVICHAQUIFER fetkovich_2d"
```

Comment:
I noticed the grammar here is not that consistent.  For each entry in `tests`, should we have four elements or three. Above this line, there are four elements. below this line, there are three. Looking at usage at the bottom, it should contain 4 elements for each entry? @akva2